src/game_window.py

```diff
@@ -0,0 +1,30 @@
+import pygame
+import src.common_constants as common_constants
+
+class GameWindow():
+    def __init__(self, title):
+        self.width = common_constants.CommonConstants.WINDOW_WIDTH
```

Comment:
really long line! change import to `from common_constants import CommonConstants`, then you can refer to WINDOW_WITH as `CommonConstants.WINDOW_WIDTH`